### ISSUE TITLE
Feat: improve mission time with DO_DIGICAM_CONTROL and Delay at WP

### ISSUE THREAD

**Issue by JoeKae** [YarostheLaunchpadder/MissionPlanner-to-Litchi#21]
When [Mission Planner](https://ardupilot.org/planner/) is creating a survey utilizing the  [DO_DIGICAM_CONTROL](https://ardupilot.org/copter/docs/mission-command-list.html#do-digicam-control) Trigger Method and a `Delay at WP` > 0, then [Mission Planner](https://ardupilot.org/planner/) is not only creating a delay at photo [waypoints](https://ardupilot.org/copter/docs/mission-command-list.html#waypoint) but at all [waypoints](https://ardupilot.org/copter/docs/mission-command-list.html#waypoint). 

While this might be desirable in some cases, it usually is not. Removing the delay from non photo [waypoints](https://ardupilot.org/copter/docs/mission-command-list.html#waypoint) could improve the mission time.

I suggest a checkbox that triggers removing those unnecessary delays.
Before adding this, there should be the new gui.

**Comment by YarostheLaunchpadder** [comment]
That would be great. But the UI will need to be done first.
I will try to continue working on the UI and checking the PySide6 docs, because I never have done a UI with this library.

**Comment by JoeKae** [comment]
> That would be great. But the UI will need to be done first.

I agree, when the UI is ready, it will be possible to set various options per file.
